spaceship.zsh

```diff
@@ -178,6 +179,11 @@ spaceship_ps2() {
 prompt_spaceship_setup() {
   autoload -Uz vcs_info
   autoload -Uz add-zsh-hook
+  if spaceship::exists async; then
+    autoload -Uz async && async
+  else
+    source $SPACESHIP_ROOT/async.zsh && async_init
```

Comment:
use `&& async_init` or `&& async` in both `if` and `else`, for consistency 😛

Reply:
`async_init` is [required only](https://github.com/mafredri/zsh-async#async_init) if `async` unavailable on `$fpath`.

Reply:
I think `async` is just a beautiful alias over `async_init`, we can use one or another regardless of how it was loaded:

https://github.com/mafredri/zsh-async/blob/5248bd85bf2c522ebfa2b8d18ee7bc736d10c791/async.zsh#L543-L545